src/stores/utilities/usedAliquot.js

```diff
@@ -31,6 +36,8 @@ function createUsedAliquot(attributes) {
     ...defaultUsedAliquotAttributes,
     ...attributes,
     tag_id: (attributes && (attributes.tag || attributes.tag_id)) ?? null,
+    volume: initialiseToAvailableVolume ? availableVolume : formatValue(attributes?.volume),
```

Comment:
Discussed in previous comments, I think we should pass in available_volume and volume in attributes and set the volume upfront instead of mutating the values in here.